devtools/datasette/fly/run.sh

```diff
@@ -8,5 +8,5 @@ ls
 mv all_dbs.tar.zst /data
 zstd -f -d /data/all_dbs.tar.zst -o /data/all_dbs.tar
 tar -xf /data/all_dbs.tar --directory /data
-datasette serve --host 0.0.0.0 ${DATABASES} --cors --inspect-file inspect-data.json --metadata metadata.yml --setting sql_time_limit_ms 5000 --port $DATASETTE_PORT > /dev/null &
+datasette serve --host 0.0.0.0 "${DATABASES}" --cors --inspect-file inspect-data.json --metadata metadata.yml --setting sql_time_limit_ms 5000 --port "$DATASETTE_PORT" > /dev/null &
```

Comment:
maybe: drop this file entirely, we're done with datasette

Reply:
We have #4481 (Shut Down Datasette) as an issue in the current sprint so this should all go away soon.